cernopendata/modules/fixtures/data/records/cms-tools-nanoaod-outreach-higgsfourleptons.json

```diff
@@ -0,0 +1,92 @@
ts at the CMS experiment from 2012 with the goal to study decays of a Higgs boson into four leptons, more precisely pairs of electrons or muons. The analysis follows loosely <a href=\"https://www.sciencedirect.com/science/article/pii/S0370269312008581\">the official CMS analysis published in 2012</a>.</p><p>This analysis studies the process $H\\rightarrow ZZ\\rightarrow 4\\ell$, the Higgs boson decaying via two Z bosons into pairs of leptons. We consider here only electrons and muons, which can be directly detected with the CMS detector. The exact same final state is present in events with two Z bosons from quark annihilation, which is the most prominent background process. It should be noted that this analysis takes only this background process into account and neglects the contribution of all other minor background processes. The four leptons can origin from three combinations of lepton pairs: two electron pairs, two muon pairs, or one electron and one muon pair. The analysis is designed to produce the plots in each of these channels but also combines them to a final result, see the example plots below. Further information about the processing steps can be found in the commented source code, <a href=\"https://www.sciencedirect.com/science/article/pii/S0370269312008581\">the official CMS publication</a> and <a href=\"/record/5500\">the similar Open Data analysis using the more complex AOD datasets</a>.</p><p>The plot below on the left-hand side shows one of the results of this Open Data analysis, whereas the plot on the right-hand side is taken from <a href=\"https://www.sciencedirect.com/science/article/pii/S0370269312008581\">the official CMS analysis published in 2012</a>. Both analyses have comparable statistics but the analyzed data is just partially the same. Further, this Open Data analysis neglects for simplicity the minor background processes.</p><p><img src=\"https://www.github.com/cms-opendata-analyses/HiggsToFourLeptonsNanoAODOutreachAnalysis/blob/master/plots/combined_Higgs_mass.png?raw=true\" width=50% align=\"middle\"><img src=\"http://cms-results.web.cern.ch/cms-results/public-results/publications/HIG-12-028/CMS-HIG-12-028_Figure_004.png\" width=50% align=\"middle\"></p><p>The analysis code of this analysis is designed so that it is easy for you to investigate not just the mass of the Higgs boson but also other features of the data. As an example, the plots below show the reconstructed mass of the two Z bosons, which are the direct decay products of the Higgs boson before those decay again instantly in the analyzed final state particles consisting of electron and muon pairs.</p><p><img src=\"https://www.github.com/cms-opendata-analyses/HiggsToFourLeptonsNanoAODOutreachAnalysis/blob/master/plots/combined_Z1_mass.png?raw=true\" width=50% align=\"middle\"><img src=\"https://www.github.com/cms-opendata-analyses/HiggsToFourLeptonsNanoAODOutreachAnalysis/blob/master/plots/combined_Z2_mass.png?raw=true\" width=50% align=\"middle\"></p>"
```

Comment:
The plots are not showing properly. The links are correct and I can access them with those direct links. @tiborsimko Is there something additional to be done to have them display or is it just a feature of preview?

Reply:
Update, the plots show fine.